module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -369,10 +328,8 @@ public function getSavedData(
      * Add/update a resource in the user's account.
      *
      * @param \VuFind\Db\Row\Resource $resource        The resource to add/update
-     * @param \VuFind\Db\Row\UserList $list            The list to store the resource
-     * in.
-     * @param array                   $tagArray        An array of tags to associate
-     * with the resource.
+     * @param int                     $list            The list to store the resource in.
```

Comment:
This isn't an integer anymore, is it? I think it's a UserList entity now. I realize all this code is going to be further refactored soon, but getting the types straightened out (and maybe adding typehints to the arguments) might make the next round of refactoring a little easier, so I don't think it's a waste of time.

Reply:
Wait, never mind, I'm wrong, this is an integer. Sorry!